src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/AsyncLazy`1.cs

```diff
@@ -576,7 +576,7 @@ public void CompleteFromTask(Task<T> task)
                 }
                 else if (task.IsFaulted)
                 {
-                    this.TrySetException(task.Exception);
+                    this.TrySetException(task.Exception!);
```

Comment:
Ah, if only we had the compiler support for property tracking like this yet... :-(